### DISCUSSION TITLE
[Feature] Notification for lookback feature

### DISCUSSION THREAD

**Discussion by DITC101** [immich-app/immich#11427] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I love the lookback feature of Immich to see what was happening on the same day in the past. Would it be possible to get a daily notification if there are pictures from previous years on that same day? Maybe you could even make the hour configurable (e.g. every day at 11:00. 

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#3981

**Comment by DITC101** [comment]
Sorry. I missed that one.
